Kumascript unit tests broken

It looks like #2276 broke the KS unit tests.

```
FAIL kumascript/tests/macros/spec2.test.js
  ● test "spec2" › WD (ja)

    expect(received).toEqual(expected) // deep equality

    Expected: "<span class=\"spec-wd\">草案</span>"
    Received: "<span class=\"spec-cr\">勧告候補</span>"

      22 | assert.eventually = {
      23 |   async equal(x, y) {
    > 24 |     expect(await x).toEqual(y);
         |                     ^
      25 |   },
      26 | };
      27 | 

      at Object.equal (kumascript/tests/macros/utils.js:24:21)

PASS kumascript/tests/macros/EmbedTest262ReportResultsTable.test.js
PASS kumascript/tests/macros/wiki-api.test.js
PASS kumascript/tests/macros/specdata.test.js
PASS kumascript/tests/macros/Compat.test.js
PASS kumascript/tests/macros/firefoxsidebar.test.js
PASS kumascript/tests/macros/toolssidebar.test.js
PASS kumascript/tests/macros/gamessidebar.test.js
PASS kumascript/tests/macros/mdnsidebar.test.js
PASS kumascript/tests/macros/utils.test.js
PASS kumascript/tests/macros.test.js
PASS kumascript/tests/macros/XULElement.test.js
PASS kumascript/tests/macros/svgattr.test.js
PASS kumascript/tests/macros/TemplateLink.test.js

Summary of all failing tests
FAIL kumascript/tests/macros/spec2.test.js
  ● test "spec2" › WD (ja)

    expect(received).toEqual(expected) // deep equality

    Expected: "<span class=\"spec-wd\">草案</span>"
    Received: "<span class=\"spec-cr\">勧告候補</span>"

      22 | assert.eventually = {
      23 |   async equal(x, y) {
    > 24 |     expect(await x).toEqual(y);
         |                     ^
      25 |   },
      26 | };
      27 | 

      at Object.equal (kumascript/tests/macros/utils.js:24:21)


Test Suites: 1 failed, 36 passed, 37 total
Tests:       1 failed, 603 passed, 604 total
Snapshots:   0 total
Time:        15.115 s
Ran all test suites matching /kumascript/i.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

--- a/kumascript/tests/macros/spec2.test.js
+++ b/kumascript/tests/macros/spec2.test.js
@@ -21,7 +21,7 @@ describeMacro("spec2", function () {
     macro.ctx.env.locale = "ja";
     return assert.eventually.equal(
       macro.call("CSS3 Box"),
-      '<span class="spec-wd">草案</span>'
+      '<span class="spec-cr">勧告候補</span>'
     );
   });
   itMacro("REC (de)", function (macro) {

